docs/user-guide/quickstart.md

```diff
@@ -1,5 +1,121 @@
-# Quickstart
+# VirtualShip Quickstart Guide 🚢
+
+Welcome to this Quickstart to using VirtualShip. In this guide we will conduct a virtual expedition in the North Sea. Note, however, that you can plan your own expedition anywhere in the global ocean and conduct whatever measurements you wish!
+
+This guide is intended to give a basic overview of how to plan, initialise and execute a virtual expedition. Data post-processing, analysis and visualisation advice is provided in other sections of the documentation (see [Output](#output) section).
+
+## Cruise planning
+
+> [!NOTE]
+> This section describes the _custom_ cruise planning procedure. There is also an option to proceed without your own cruise plan and you can instead use an example route, schedule and selection of measurements (see [Initialise the expedition](#initialise-the-expedition) for more details).
```

Comment:
I try to avoid the the word 'cruise' within the VirtualShip. While it's a common phrase in oceanography, it means something completely different outside of oceanography and paints a wrong picture of what seagoing work is. So I prefer to stick with 'expedition'
```suggestion
> This section describes the _custom_ expedition planning procedure. There is also an option to proceed without your own expedition plan and you can instead use an example route, schedule and selection of measurements (see [Initialise the expedition](#initialise-the-expedition) for more details).
```